src/main/java/math/Fraction.java

```diff
@@ -182,8 +182,11 @@ public Fraction multiply(Fraction fraction) {
      * @return a fraction that represents the reciprocal of this fraction
      */
     public Fraction reciprocal() {
-        // TODO (X0Q03E)
-        return null;
+        if (numerator<0){
```

Comment:
Note that the `if` statement here may be remove from here without breaking the method.